src/rlookup.c

```diff
@@ -264,6 +264,42 @@ static RSValue *hvalToValue(RedisModuleString *src, RLookupCoerceType type) {
   }
 }
 
+static RSValue *jsonValToValue(RedisModuleCtx *ctx, RedisJSON json) {
+  size_t len;
+  char *str;
+  const char *constStr;
+  RedisModuleString *rstr;
+  RSValue *rs_val;
+  long long ll = 0;
+  double dd = 0.0;
```

Comment:
No need to initialize here as we rely on `japi->get*` to either get a value or fail. If we want to initialize nonetheless, we can do it in a block just before `japi->get*`.